.gitattributes

```diff
@@ -54,5 +54,5 @@ tools/CoqMakefile.in whitespace=blank-at-eol
 # CR is desired for these Windows files.
 *.bat whitespace=cr-at-eol,blank-at-eol,tab-in-indent
 
-* eol=lf
-*.bat eol=crlf
+*.sh eol=lf
+/configure eol=lf
```

Comment:
When I saw that configure needed the eol=lf I was worried that I would have to list a bunch of files but this was unfounded. This way is definitely better.